src/utils/ledger.js

```diff
@@ -125,6 +157,61 @@ if (IPC_RENDERER) {
       }
       IPC_RENDERER.send("ledger:getSignatures", arr);
     },
+
+    /**
+     * Sign token info on ledger
+     *
+     * @param {Object} token
+     *  with uid (hex), symbol and name
+     *
+     * @memberof Ledger
+     * @inner
+     */
+    signToken(token) {
+      const data = Buffer.concat(serializeTokenInfo(token, false));
```

Comment:
I've seen in https://github.com/HathorNetwork/hathor-ledger-app/pull/38 that we will have limits on the token name and symbol lengths.

Should we add a validation for this here before sending the token to Ledger?

Or this validation already exists somewhere? (like when minting the token)

Reply:
There is a PR for the sign token modal with this validation, it's better to leave it on the component context since it can better show where/what went wrong.

Reply:
I think adding a validation here is something you should still consider.

Since it's a utils method to interact with ledger, we could use it in other places in the future besides the component, and we should make sure the validation is done every time.

Reply:
I think you're right that we should have a "centralized" validation, but I don't think this is the place to do this.

I have an idea to remove the `public/ledger.js` module into a separate lib, we could later do all validation on this module and have the callers catch validation errors. This seems better than having duplicated validations here.

Reply:
Ok, I think that makes sense.

But I just think you should create an issue explaining this idea, if there isn't one already.